code/drasil-theory/lib/Theory/Drasil/MultiDefn.hs

```diff
@@ -1,59 +1,95 @@
-{-# LANGUAGE TemplateHaskell, Rank2Types, ScopedTypeVariables, PostfixOperators  #-}
+{-# LANGUAGE PostfixOperators #-}
+{-# LANGUAGE Rank2Types #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TemplateHaskell #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE TupleSections #-}
 
 -- | Defines types and functions for creating mult-definitions.
-module Theory.Drasil.MultiDefn (
-  -- * Types
-  MultiDefn, DefiningExpr,
-  -- * Constructors
-  mkMultiDefn, mkMultiDefnForQuant, mkDefiningExpr,
-  -- * Functions
-  multiDefnGenQD, multiDefnGenQDByUID) where
-
-import Control.Lens ((^.), view, makeLenses)
+module Theory.Drasil.MultiDefn
+  ( -- * Types
+    MultiDefn,
+    DefiningExpr,
+
+    -- * Constructors
+    mkMultiDefn,
+    mkMultiDefnForQuant,
+    mkDefiningExpr,
+
+    -- * Functions
+    multiDefnGenQD,
+    multiDefnGenQDByUID,
+  )
+where
+
+import Control.Lens (makeLenses, view, (^.))
 import Data.List (union)
 import qualified Data.List.NonEmpty as NE
-
 import Language.Drasil hiding (DefiningExpr)
 import Language.Drasil.Development (showUID)
 
 -- | 'DefiningExpr' are the data that make up a (quantity) definition, namely
 --   the description, the defining (rhs) expression and the context domain(s).
 --   These are meant to be 'alternate' but equivalent definitions for a single concept.
-data DefiningExpr e = DefiningExpr {
-  _deUid  :: UID,            -- ^ UID
-  _cd     :: [UID],          -- ^ Concept domain
-  _rvDesc :: Sentence,       -- ^ Defining description/statement
-  _expr   :: e  -- ^ Defining expression
-}
+data DefiningExpr e = DefiningExpr
```

Comment:
I actually preferred the previous layout for this.

Reply:
I think I fixed it now.